.s2i/bin/assemble

```diff
@@ -8,15 +8,12 @@
 #
 
 # If the 'billrainford/tensorflowapp-sample' assemble script is executed with the '-h' flag, print the usage.
-set -ex
+echo "This is start of assemble script"
 
-echo "---> Running as UID:"
```

Comment:
I would highly advise leaving lines 12-16 in the sample as it made it a lot easier to see and debug whats going on and fore the purposes of demoing this provided useful output.

Reply:
Done